### PR TITLE
Issue #SB-19927 fix: Added encryption and decryption for sso flow.

### PR DESCRIPTION
**Pull request by rajeshkumaravel**

### CODE REVIEW

File: src/app/routes/accountRecoveryRoute.js [hunk 1]
```diff
@@ -73,14 +73,18 @@ module.exports = (app) => {
...
             }
             if (req.body.request.userId) {
-              encrypt['id'] = req.body.request.userId
+              _encrypt['id'] = req.body.request.userId
             }
             var timeInMin = 5;
-            var validator = encriptWithTime(encrypt, timeInMin);
+            var validator = encriptWithTime(_encrypt, timeInMin);
+            const reqType = req.body.request.type;
+            const dataToEncrypt = {};
```
Comment (deveshMantra) [inline src/app/routes/accountRecoveryRoute.js:85]: shall we be using expirating time as we did for other places ?

File: src/app/routes/accountRecoveryRoute.js [hunk 2]
```diff
@@ -73,14 +73,18 @@ module.exports = (app) => {
...
+            dataToEncrypt[reqType] = req.body.request.key;
+            req.session.otpEncryptedInfo = encrypt(JSON.stringify(dataToEncrypt));
```
Comment (deveshMantra) [inline src/app/routes/accountRecoveryRoute.js:87]: is two times encryption is required ?
can we include if condition like for user id ?

File: src/app/routes/accountRecoveryRoute.js (hunk 1 above)
Comment (rajeshkumaravel) [reply src/app/routes/accountRecoveryRoute.js:85]: No.
We are not using time expiry.

File: src/app/routes/accountRecoveryRoute.js (hunk 2 above)
Comment (rajeshkumaravel) [reply src/app/routes/accountRecoveryRoute.js:87]: `if` condition on what ?
type is always available in verify OTP route.

**Pull request merged**